src/build-meta.ts

```diff
@@ -1,3 +1,3 @@
-export const REVISION = 'd5a1a2f'
-// eslint-disable-next-line @typescript-eslint/no-loss-of-precision, prettier/prettier
-export const TIMESTAMP = 1719122331.228639166
+export const REVISION = ' 1070b50' 
```

Comment:
Same. Some extra spaces.

Reply:
I think it's due to `git describe --tags --abbrev=0` not returning anything, which means the tag list is not pulled successfully. It should be resolved when built with the tag list available.

Reply:
so should I do anything about that?

Reply:
Maybe just manually put `v1.2.5` at this point? Or try `git fetch --tags` before building again

Reply:
If the script is correct, nothing needs to be changed.